integration/tools/validation/src/main/java/alluxio/cli/UnderFileSystemContractTest.java

```diff
@@ -81,6 +80,15 @@ public UnderFileSystemContractTest(String path, InstancedConfiguration conf) {
     mConf = conf;
   }
 
+  /**
```

Comment:
This class is now split into two:
1. The real work logic is here
1. The trigger is `UfsTests` class in `shell/` module